Shell/AnswerLiteralManager.hpp

```diff
@@ -162,6 +162,17 @@ class SynthesisALManager : public AnswerLiteralManager
     DHMap<unsigned, unsigned> _condFnToPred;
   };
 
+  class ComputableSelfTransformer : public BottomUpTermTransformer {
```

Comment:
Is this used anywhere? I don't see it's implementation in `AnswerLiteralManager.cpp`.